### DISCUSSION TITLE
run dp with multiple node

### DISCUSSION THREAD

**Discussion by jingganglan** [deepmodeling/deepmd-kit#131] [Q&A]
Hi,

I am trying to run dp with multiple nodes, each node has 1 gpu.
I am wondering if this is possible?

export OMP_NUM_THREADS=$SLURM_CPUS_PER_TASK
export CRAY_CUDA_MPS=1
export CUDA_VISIBLE_DEVICES=0

srun -N dp train water_se_a.json > log

this run script doesn't work. dp will only use one GPU

**Comment by jameswind** [comment]
Hi, right now the DeePMD-kit code does not support multi-GPU training. It
only supports multi-GPU MD.

**Reply by tonystarkiss** [reply]
Does it still not support multi gpu training now? 
I find it training on one GPU default, I don't know how to use multi GPU accelerate training?

**Reply by njzjz** [reply]
Now it's under development in the devel branch.

**Reply by tonystarkiss** [reply]
the multi GPU training  feature is not included in v2.0.0? When will it release ?

**Comment by jingganglan** [comment]
thanks
